config/main.py

```diff
@@ -8053,5 +8053,82 @@ def max_sessions(max_sessions):
                         {'max_sessions': max_sessions})
 
 
+#
+# 'logrotate' group ('config logrotate ...')
+#
+@config.group(invoke_without_command=True)
+@click.pass_context
+@click.argument('file', required=True, type=click.Choice(['syslog', 'debug']))
+def logrotate(ctx, file):
+    """Configuring logrotate"""
+    # If invoking subcomand, no need to do anything
+    if ctx.invoked_subcommand is not None:
+        return
+
+    config_db = ConfigDBConnector()
+    config_db.connect()
+    config_db.mod_entry(swsscommon.CFG_LOGGING_TABLE_NAME, file, None)
+
+
+@logrotate.command()
+@click.pass_context
+@click.argument('disk_percentage', metavar='<disk-percentage>',
+                required=True,  type=float)
+def disk_percentage(ctx, disk_percentage):
+    """Configuring logrotate disk-precentage"""
+    file = ctx.parent.params.get('file')
+    if 0 > disk_percentage > 100:
```

Comment:
>0 > disk_percentage > 100 [](http://example.com/codeflow?start=7&length=25)

Does it actually prevent invalid values here, should it be `0 > disk_percentage or disk_percentage > 100`?